src/libraries/Directory.Build.props

```diff
@@ -79,6 +81,13 @@
         <PackageTargetRuntime>osx</PackageTargetRuntime>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetFrameworkSuffix)' == 'iOS'">
+      <PropertyGroup>
+        <TargetsUnix>true</TargetsUnix>
+        <TargetsiOS>true</TargetsiOS>
```

Comment:
Hmm this looks weird to read, but I think we should be consistent across all places with use of `iOS` so I think it is fine as is.